tests/cases/fourslash/jsDocGoToImplementation.ts

```diff
@@ -0,0 +1,10 @@
+///<reference path="fourslash.ts" />
+
+/////**
+//// * @param /**/[|foo|] I pity the foo
+//// */
+////function f([|foo|]: number) {
+////    return foo;
```

Comment:
D'oh, shouldn't have left this in. We don't support goto-implementation for parameters anyway.